src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/services/MIReplicaReportingServiceTest.java

```diff
@@ -95,6 +98,94 @@ void should_save_task_and_get_task_from_reportable_task() {
                 });
     }
 
+    @ParameterizedTest
+    @CsvSource(value = {
+        "ADD",
+        "UPDATE"
+    })
+    void should_save_task_and_get_task_from_replica_tables_with_new_columns(String operation) {
+        TaskResource taskResource;
```

Comment:
Are there some negative scenarios to consider?  E.g. if you pass in null/empty values to the new fields?

Reply:
As all the existing tests don't have these new fields, it automatically covers null/empty. 

We have come up with addMissingParameters(taskResource) method only for these new tests.

Reply:
Not quite .. they won't have an assert for a null or empty field in these new fields unless they're using some reflective lib (not sure they are).

Reply:
Also, added tests to cover NULLs.